docs/InputsOutputs.rst

```diff
@@ -14,10 +14,24 @@ Input files
 ===========
 
 The UPP requires the following input files:
+ - The model forecast file
  - The itag namelist file
  - The GRIB2 control file
  - Additional data files (e.g. lookup tables, coefficient files for satellite)
 
+--------------
+Model Forecast
+--------------
+
+The UPP ingests FV3 write component files in netCDF and parallel netCDF format.
+
+The table below is a list of the unified model variables available from the UFS applications. Whether a
```

Comment:
I'm wondering if we should use "FV3 model core" instead of "UFS applications" here?

Reply:
Sounds good - changed